staging/src/k8s.io/apiserver/pkg/util/proxy/dial_test.go

```diff
@@ -96,7 +96,10 @@ func TestDialURL(t *testing.T) {
 			ts.TLS = &tls.Config{Certificates: []tls.Certificate{cert}}
 			ts.StartTLS()
 
-			tlsConfigCopy := tc.TLSConfig
+			// Make a copy of the config
+			tlsConfigCopy := tc.TLSConfig.Clone()
+			// Clone() mutates the receiver (!), so also call it on the copy
```

Comment:
> or do you mean the serverInitOnce.Do call

exactly, which flips a bit in the sync.Once, and fails a reflect.DeepEqual check